pkgs/build-support/php/build-php.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenvNoCC, lib, writeTextDir, php, makeBinaryWrapper }:
+
+let
+  buildPhpProjectOverride = finalAttrs: previousAttrs:
+
+  let
+    phpDrv = finalAttrs.php or php;
+    composer = finalAttrs.composer or phpDrv.packages.composer;
+    composerLock = finalAttrs.composerLock or null;
+  in {
+    nativeBuildInputs = [
```

Comment:
For `wp-cli`, you will want to add `previousAttrs.nativerBuildInputs` to this.

Reply:
Thanks, fixed!